src/main/java/ui/UI.java

```diff
@@ -540,4 +546,42 @@ public boolean isWindowMinimized() {
     public boolean isWindowFocused() {
         return mainStage.isFocused();
     }
+
+    /**
+     * Warns user if the Java runtime version is lower than HT's requirement
+     */
+    private void warnIfJavaVersionOutdated() {
+        JavaVersion requiredVersion, runtimeVersion;
+        String javaRuntimeVersionString = System.getProperty("java.runtime.version");
+
+        try {
+            requiredVersion = JavaVersion.fromString(REQUIRED_JAVA_VERSION);
+        } catch (IllegalArgumentException e) {
+            logger.error("Required Java Version string cannot be parsed. This should have been covered by test.");
+            return;
+        }
```

Comment:
This is not the user's java version. There is no way this exception happen right? If it did happen, we want to catch it during testing.